extras/systemd/foreman.service

```diff
@@ -2,13 +2,14 @@
 Description=Foreman
 Documentation=https://theforeman.org
 After=network.target remote-fs.target nss-lookup.target
+Requires=foreman.socket
 
 [Service]
 Type=simple
 User=foreman
 TimeoutSec=300
 WorkingDirectory=/usr/share/foreman
-ExecStart=/usr/bin/rails server --environment $FOREMAN_ENV --port $FOREMAN_PORT --binding $FOREMAN_BIND
+ExecStart=/usr/share/foreman/bin/rails server --environment $FOREMAN_ENV --port $FOREMAN_PORT --binding $FOREMAN_BIND
```

Comment:
Why is this change required?

Reply:
It was needed to avoid a Bad File Descriptor error that I couldn't figure out. In packaging we also change it to wrap it in `scl enable tfm` which breaks socket activation as well. This script has `/usr/bin/tfm-ruby` as the shebang in RPM packaging and works.